src/pages/login/oauth/LoginOAuth.vue.scss

```diff
@@ -0,0 +1,10 @@
+.page--oauth {
+  width: 100%;
```

Comment:
You can use `@extend %absolute-view` to make the view absolute and correctly positioned, no need to declare those properties on the selector